netebpfext/net_ebpf_ext_hook_provider.c

```diff
@@ -182,11 +162,14 @@ _net_ebpf_extension_detach_client_completion(_In_ DEVICE_OBJECT* device_object,
     NET_EBPF_EXT_LOG_EXIT();
 }
 
-bool
+void
```

Comment:
is this the right approach? There can be genuine cases where a callout might be invoked after `ExWaitForRundownProtectionRelease` is called?

Reply:
Until explicitly proven otherwise, I'd much rather treat this as a dangerous bug as it becomes a very slippery slope where distinguishing genuine exceptions from bogus ones might become too complicated.

IMO I'd rather prefer a re-design to avoid such instances.  Thoughts?

Reply:
Per offline discussion, moved __fastfail to a higher level macro, to leave open the possibility of using the old call signature 'as-is' in other non-filter_context related use cases.